### PR TITLE
Page detail feature/add film detail

### PR DESCRIPTION
**Pull request by brewost**
Add in detail page of film. ID currently hardcoded, it will be needed to be passed as params once possible. Comments and note feature to be added.

### CODE REVIEW

**Review by alexandreg67 — DISMISSED** [review]

**Review by Ludovicscelles — DISMISSED** [review]

File: client/src/pages/PageFilmDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          ></div>
+          <img
+            src={`${basePosterUrl}${film.posterPath}`}
+            alt={`Film poster of ${film.title}`}
+            className="w-full h-full object-cover rounded-2xl shadow-lg border-2 border-white relative"
+            style={{ zIndex: 1 }}
+          />
+        </div>
+      )}
+      <div className="w-[70%] space-y-4 border border-bloodRed rounded p-6 mt-6 text-white text-left">
```
Comment (jujuck) [inline client/src/pages/PageFilmDetail.tsx]: Une balise article ou section

File: client/src/pages/PageFilmDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  if (error) return <p className="text-red-500">Error: {error.message}</p>;
+
+  const film = data?.getFilmById;
+
+  if (!film) return <p>Film not found</p>;
+
+  return (
+    <main className="flex flex-col items-center p-4 relative">
+      {film.posterPath && (
+        <div className="relative w-[30rem] h-[45rem] mt-5 mb-10">
```
Comment (jujuck) [inline client/src/pages/PageFilmDetail.tsx]: une balise section

File: client/src/pages/PageFilmDetail.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            alt={`Film poster of ${film.title}`}
+            className="w-full h-full object-cover rounded-2xl shadow-lg border-2 border-white relative"
+            style={{ zIndex: 1 }}
+          />
+        </div>
+      )}
+      <div className="w-[70%] space-y-4 border border-bloodRed rounded p-6 mt-6 text-white text-left">
+        <h1 className="text-2xl font-bold mb-4">{film.title}</h1>
+        <p>
+          <strong className="font-semibold">Overview:</strong> {film.overview}
```
Comment (jujuck) [inline client/src/pages/PageFilmDetail.tsx]: Mets en place un padding ou margin pour simuler l'espace

**Review by jujuck — CHANGES_REQUESTED** [review]

**Review by alexandreg67 — DISMISSED** [review]

**Review by Ludovicscelles — DISMISSED** [review]

File: client/src/pages/PageFilmDetail.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+          />
+        </section>
+      )}
+      <article className="w-[70%] space-y-4 border border-bloodRed rounded p-6 mt-6 text-white text-left">
+        <header>
+          <h1 className="text-2xl font-bold mb-4">{film.title}</h1>
+        </header>
+        <section>
+          <p className="mb-4">
+            <strong className="font-semibold mr-2">Synopsis:</strong>{" "}
```
Comment (jujuck) [inline client/src/pages/PageFilmDetail.tsx]: Supprimer les espaces morts

File: client/src/main.tsx [hunk 5]
```diff
@@ -16,6 +18,10 @@ const router = createBrowserRouter([
         path: "/",
         element: <HomePage />,
       },
+      {
+        path: "/film/:id",
```
Comment (jujuck) [inline client/src/main.tsx]: /films/:id

**Review by jujuck — CHANGES_REQUESTED** [review]

**Review by alexandreg67 — APPROVED** [review]

**Review by YOUNS28100 — APPROVED** [review]

**Review by jujuck — APPROVED** [review]

**Pull request merged**